school-library/person/persons.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', _parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+  end
+
+  def of_age?
+    return true if @age >= 18
+
+    false
+  end
+
+  def can_use_services?
+    return true if @age >= 18 || parent_permission
+
+    false
+  end
```

Comment:
- [ ] I think it'll be good to use the `is_of_age?` method in the `can_use_services?` method for checking if the person is of age to avoid code repetition.